nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/MockCacheService.groovy

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
I used IntelliJ's Groovy convert to Java function to get what you see below. Just use this for now since the compiler plugin appears to not be behaving itself here.

```
package org.apache.nifi.processors.standard;

import org.apache.nifi.controller.AbstractControllerService;
import org.apache.nifi.distributed.cache.client.Deserializer;
import org.apache.nifi.distributed.cache.client.DistributedMapCacheClient;
import org.apache.nifi.distributed.cache.client.Serializer;

import java.io.IOException;
import java.util.LinkedHashMap;

public class MockCacheService extends AbstractControllerService implements DistributedMapCacheClient {
    @Override
    public <K, V> boolean putIfAbsent(K k, V v, Serializer<K> serializer, Serializer<V> serializer1) throws IOException {
        boolean retVal = map.containsKey(k);
        if (retVal) {
            return false;
        } else {
            map.put(k, v);
            return true;
        }

    }

    @Override
    public <K, V> V getAndPutIfAbsent(K k, V v, Serializer<K> serializer, Serializer<V> serializer1, Deserializer<V> deserializer) throws IOException {
        return null;
    }

    @Override
    public <K> boolean containsKey(K k, Serializer<K> serializer) throws IOException {
        return map.containsKey(k);
    }

    @Override
    public <K, V> void put(K k, V v, Serializer<K> serializer, Serializer<V> serializer1) throws IOException {

    }

    @Override
    public <K, V> V get(K k, Serializer<K> serializer, Deserializer<V> deserializer) throws IOException {
        return null;
    }

    @Override
    public void close() throws IOException {

    }

    @Override
    public <K> boolean remove(K k, Serializer<K> serializer) throws IOException {
        return false;
    }

    @Override
    public long removeByPattern(String s) throws IOException {
        return 0;
    }

    public void assertContains(String key, String value) {
        assert map.containsKey(key) && map.get(key).equals(value);
    }

    public LinkedHashMap<Object, Object> getMap() {
        return map;
    }

    public void setMap(LinkedHashMap<Object, Object> map) {
        this.map = map;
    }

    private LinkedHashMap<Object, Object> map = new LinkedHashMap<Object, Object>();
}
```

Reply:
I moved the groovy implementation to the Groovy test folder.